Source/SwiftLintBuiltInRules/Rules/Idiomatic/RedundantTypeAnnotationRule.swift

```diff
@@ -234,14 +234,14 @@ private extension TypeAnnotationSyntax {
         // Consider the type annotation redundant if `considerLiteralsRedundant` is true and the expression is one of
         // the supported compiler-inferred literals.
         if considerLiteralsRedundant {
-            return isCompilerInferredLiteral(expression: initializer)
+            return isCompilerInferredLiteral(initializer)
```

Comment:
I would actually expect this to be called outside of this method separately to avoid passing the option around.